docs/VERIFICATION_CHECKLIST.md

```
@@ -0,0 +1,346 @@
+```bash
+# 安装所有依赖
+npm install
+
+# 启动服务
+npm run dev
+```
+
+**验证点**:
+- [x] 后端启动无错误
+- [x] 前端启动无错误
+- [x] 浏览器访问 http://localhost:3000 正常
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)